hepcrawl/utils.py

```diff
@@ -73,20 +74,26 @@ def collapse_initials(name):
     return name
 
 
-def split_fullname(author, surname_first=True):
+def split_fullname(author, switch_name_order=False):
     """Split an author name to surname and given names.
 
-    It accepts author strings with and without comma separation
-    and surname can be first or last. Note that multi-part surnames are incorrectly
-    detected in strings without comma separation.
+    It accepts author strings with and without comma separation.
+    As default surname is first in case of comma separation, otherwise last.
+    Note that multi-part surnames are incorrectly detected in strings without comma separation.
     """
     if not author:
         return "", ""
 
     if "," in author:
         fullname = [n.strip() for n in author.split(',')]
+        surname_first = True
     else:
         fullname = [n.strip() for n in author.split()]
+        surname_first = False
+
+#    surname_first = not switch_name_order == surname_first
```

Comment:
I prefer to spell this out (the following 2 lines), others prefer arithmetics.
Whatever you want.